### DISCUSSION TITLE
Single type localization problem

### DISCUSSION THREAD

**Discussion by NevenaKirova** [gatsby-uc/plugins#374] [Q&A]
I have a problem getting all locales for singleTypes. 

	{
		singularName: 'about',
		queryParams: {
			populate: '*',
		},
		pluginOptions: {
			i18n: {
				locale: 'all',
			},
		},
	},

this is an exapmle from gatsby.config and i only get the default one.

**Comment by moonmeister** [comment]
cc: @laurenskling

**Reply by laurenskling** [reply]
I don't use i18n myself, so I can't really help with this
